README.md

```diff
@@ -1 +1,161 @@
-# bls-solidity
\ No newline at end of file
+# bls-solidity [![TravisCI](https://travis-ci.com/witnet/bls-solidity.svg?branch=master)](https://travis-ci.com/witnet/bls-solidity)
+
+`bls-solidity` is an open source fast and effective implementation of BN256 operations written in Solidity. More precisely, this library implements all the necessary curve arithmetic to perform BLS signatures. Currently only the Elliptic Curve (EC) `bn256` is supported.
```

Comment:
Maybe add "256- bit Barreto-Naehrig"

Reply:
Should we add that one of the reasons is because this curve is the one used in the pre-compiled EVM functions?

Reply:
yes